src/app/shared/shared.module.ts

```diff
@@ -31,6 +32,7 @@ import { NavigationStackUtils } from './utils/navigation-stack-utils';
 import { ContactSelectTableRowsComponent } from './components/contact-select-table-rows/contact-select-table-rows.component';
 import { ExtensionComponent } from './components/extension/extension.component';
 import { ListResolver } from './resolvers/list-resolver.service';
+import { from } from 'rxjs';
```

Comment:
I think it might have auto-imported from you editor when you typed the word 'from'